src/main/kotlin/domain/MineBoard.kt

```diff
@@ -0,0 +1,3 @@
+package domain
+
+class MineBoard(private val mineBoard: HashMap<Position, Block>)
```

Comment:
인터페이스를 타입으로 사용하는 습관을 길러 두면 프로그램이 훨씬 유연해질 거예요. 인터페이스와 다형성에 대해 알아보고 아래의 글도 참고하면 좋을 것 같아요.
https://stackoverflow.com/questions/11683044/return-arraylist-or-list/11683073

```suggestion
class MineBoard(private val mineBoard: Map<Position, Block>)
```